changelogs/unreleased/gh-9644-verbose-uri-error.md

```diff
@@ -2,3 +2,5 @@
 
 * Added a subject URI to the error that is thrown on configuration
   verification (gh-9644).
+* Added a warning with the skipped (unsuitable) URI to replicaset
+  and sharding configuration (gh-9644).
```

Comment:
Nit: It is ~30 commit before the master. I would rebase the change before the push just in case.

Reply:
Done.